packaging/centos-7/build-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alanfranz/fwd-centos-7:latest
```

Comment:
Ah ok. Mind adding that as a comment? That's OK to keep it then. I just wasn't sure *why*.